README.md

```diff
@@ -4,12 +4,35 @@ A tool to quickly clean up your email inbox!
 
 ## Demo
 
-https://github.com/user-attachments/assets/b52220a0-b69e-4dcb-8eb2-f1bbe472d536
+<https://github.com/user-attachments/assets/b52220a0-b69e-4dcb-8eb2-f1bbe472d536>
 
 ## How to run
 
-1. install requirements: `pip install -r requirements.txt`
-2. run streamlit app: `streamlit run main.py`
+1. install uv (if you don't have it)
+
+   ```bash
+   curl -LsSf https://astral.sh/uv/install.sh | sh
+   ```
+
+   Make sure you follow the instructions once the installation completes.
+
+1. install dependencies with uv
+
+   ```bash
+   uv sync
+   ```
+
+1. activate the virtual environment
+
+   ```bash
+   source .venv/bin/activate
+   ```
+
+1. run streamlit app:
+
+   ```bash
+   streamlit run main.py`
```

Comment:
Let's use `uv run` so that we don't have to worry about activating the virtual env. with that we can remove the above step from readme